packages/expo-module-template/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@
 
 ### 🛠 Breaking changes
 
+- Move android files form `kotlin` to `java` directory in order to conform to our build scripts ([#8746](https://github.com/expo/expo/pull/8746) by [@mczernek](https://github.com/mczernek))
+
 ### 🎉 New features
 
+- Add template native test file along with suggested test dependencies ([#8746](https://github.com/expo/expo/pull/8746) by [@mczernek](https://github.com/mczernek))
+
```

Comment:
I think we should keep this changelog empty (with just headers) – or at least make it empty when creating a new module from it, but I can take care of that later 😉

Reply:
Cleaned, will need to update publish script to avoid this module.

However, since this package is not really being used by users, I guess we can skip it in here.